db/seeds.rb

```diff
@@ -715,13 +715,26 @@
   nick = Participant.create({ andrewid: 'nharper', phone_number: 1713435788, user: nick_user })
   Membership.create({ organization: dtd_org, participant: nick })
 
-  puts 'Old Stuff...'
- tool = Tool.create({ name: 'Hammer', barcode: 7, description: 'it\'s a hammer' })
+  # Uncomment the line below and comment the creation of tool types and tools (so there is no conflicts)
+  #   to get a bunch of tools seeded into the database
```

Comment:
I would say we should always generate tool types regardless of environment (which we do for all other types such as shift types, etc.). I think the only option should be between 5 random tools and all the tools.